engines/python/setup/djl_python/tensorrt_llm.py

```diff
@@ -97,7 +108,37 @@ def parse_input(
 
         return input_data, input_size, parameters, errors, batch
 
-    def inference(self, inputs: Input) -> Output:
+    def _get_config(self, properties):
+        model_path = self.trt_configs.model_id_or_path
+        if not os.path.isfile(os.path.join(model_path, 'config.json')):
+            model_path = toolkit_utils.get_python_backend_engine_path(
+                model_path, properties)
+            if not os.path.isfile(os.path.join(model_path, 'config.json')):
+                raise ValueError(
+                    f"Could not find config.json in {self.trt_configs.model_id_or_path} or"
+                    f"{model_path} for TensorRT python backend")
+
+        return AutoConfig.from_pretrained(
+            model_path, trust_remote_code=self.trt_configs.trust_remote_code)
+
+    def _load_model(self, properties):
+        if self.is_rolling_batch_enabled:
+            self.rolling_batch = TRTLLMRollingBatch(
+                self.trt_configs.model_id_or_path, properties, **properties)
+        else:
+            model_config = self._get_config(properties)
+            if model_config.model_type in self.PYTHON_BACKEND_SUPPORTED_MODELS:
+                self.model = tensorrt_llm_toolkit.init_inference(
+                    self.trt_configs.model_id_or_path,
+                    **properties,
+                    use_python_backend=True)
+            else:
+                raise ValueError(
+                    f"You cannot disable rolling batch if its not any of these models"
+                    f" {self.PYTHON_BACKEND_SUPPORTED_MODELS}. Please enable it with auto or tensorrt "
```

Comment:
I think there's no value `tensorrt` for rolling batch - https://github.com/deepjavalibrary/djl-serving/blob/3e948d8307505b3e9b59dd6a1da3143136a5fef3/engines/python/setup/djl_python/properties_manager/properties.py#L19. `trtllm`

Reply:
Done